CMakeLists.txt

```diff
@@ -62,12 +66,17 @@ set(LIB_TDNF "tdnf")
 set(LIB_TDNF_SOLV "tdnfsolv")
 set(LIB_TDNF_COMMON "common")
 set(LIB_TDNF_CLI tdnfcli)
+set(SHEBANG "#!/usr/bin/env bash")
```

Comment:
Why is this needed?

Reply:
I'm using this in bin/tdnf-automatic.in and bin/tdnf-cache-updateinfo.in

I prefer using #!/usr/bin/env bash over #!/bin/bash after I faced some trouble in MacOS

SHEBANG here is a provision for developers, if they need it, they can use it.

Also, in bin/tdnf-cache-updateinfo.in, I wanted the script to depend on cmake for some variable at least instead of straight away pushing the script to build output. It also helps in case if we want to make use of few more cmake variables in script and we can recognize easily that this is dependent on cmake. Hope this clarifies question.

Reply:
Yes it does thank you. Two points/suggestions,

1. Can you not hardcode `#!/usr/bin/env bash` in script? I think it would work both places. The reason is, having a bunch on .in files is messy (starting with syntax recognition to language type categorization). If we start doing configure for SHEBANGs then we'd end up having all shell scripts with .in suffix just so it uses the SHEBANG.
1. Would it be better to put all such configuration variables in a separate variable `tdnf-automatic-conf.sh.in` and then source them in your script at runtime? Again, just to keep the number congure_files low.